java/src/main/java/com/factset/protobuf/stach/extensions/v2/ColumnOrganizedStachExtension.java

```diff
@@ -115,6 +116,12 @@ private TableData generateTable(PackageProto.Package packageObj, String primaryT
         Map<String, MetadataItemProto.MetadataItem> metadata = primaryTable.getData().getMetadata().getItemsMap();
         for (Map.Entry<String, MetadataItemProto.MetadataItem> entry : metadata.entrySet()) {
             table.getMetadata().put(entry.getKey(), StachUtilities.valueToString(entry.getValue().getValue()));
+
+            List<Value> values = new ArrayList<>();
+            for (int i=0; i<entry.getValue().getValue().getListValue().getValuesList().size(); i++) {
```

Comment:
better to keep the entry.getValue().getValue().getListValue().getValuesList() in a seperate variable and reuse that